src/Components/NewForm/NewForm.js

```diff
@@ -47,8 +47,9 @@ class NewForm extends Component {
         })
     }
 
-    handleSubmit(){
-        const address = `${this.state.address_street} ${this.state.address_apt}, ${this.state.address_city}`
+    handleSubmit(e){
+        e.stopPropagation();
+        const address = `${this.state.address_street}${this.state.address_apt},${this.state.address_city}`
```

Comment:
This could maybe be its own util function, `formatAddress` or something. Remember, one function does one thing.